.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, windows-latest]
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
`3.6` is removed here because it has been sunset: [Link](https://discuss.python.org/t/python-3-6-rides-into-the-sunset/12964)